Source/WebKit/NetworkProcess/storage/LocalStorageManager.h

```diff
@@ -65,6 +65,8 @@ class LocalStorageManager {
     void cancelConnectToTransientLocalStorageArea(IPC::Connection::UniqueID);
     void disconnectFromStorageArea(IPC::Connection::UniqueID, StorageAreaIdentifier);
 
+    HashMap<String, String> const fetchStorageMap();
```

Comment:
`HashMap<String, String> fetchStorageMap() const`?

The `const` is at a weird location..

Reply:
Oops! Fixed.